### DISCUSSION TITLE
More examples please

### DISCUSSION THREAD

**Discussion by hsgarn** [obs-websocket-community-projects/obs-websocket-java#155] [Q&A] [answered]
I have the connection working and implemented some logic in the onReady method.  However, as a not so advanced Java programmer, I am having trouble implementing the onClose. Could someone add an example of using the onClose lifecycle event?
An example of using SetInputSettings & GetInputSettings would also be helpful.

And thanks for all the work on the new version.

**Comment by ChristopheCVB** [comment, marked as answer]
Join us on [Discord](https://discord.gg/F7spyEQWjM), we'll help you there ;)
